sycl/test-e2e/Matrix/slm_utils.hpp

```diff
@@ -22,22 +26,22 @@ slm_read_write(multi_ptr<TOperand, space, access::decorated::yes> pA,
         k2 * KCache2;
     size_t LocOffsetA = (sg.get_group_id() * (MCache2 / SGs) + i) * KCache2;
 
+    auto SrcA = pA + GlOffsetA;
+    auto DstA = tileA.template get_multi_ptr<sycl::access::decorated::no>() +
+                LocOffsetA;
+
     if (elemsPerLoadA == 2) {
-      vec<TOperand, 2> slmVecA = sg.load<2>(pA + GlOffsetA);
-      sg.store<2>(tileA.template get_multi_ptr<sycl::access::decorated::no>() +
-                      LocOffsetA,
-                  slmVecA);
+      vec<TOperand, 2> slmVecA;
+      group_load(sg, SrcA, slmVecA, properties(data_placement_striped));
```

Comment:
why did you choose here data_placement_striped instead of data_placement_blocked. Is the striped one the one that guarantees vectorization inside the SG?

Reply:
I chose it because (IIUC) that matches the behavior of the old API. I didn't use the property for scalar overloads because both data layouts are essentially the same for scalars.

Reply:
Sounds good if this is old API behavior.
It is weird then that by default, they made default placement is the blocked one.

Reply:
> Sounds good if this is old API behavior.

Old: 

> Returns: Returns N elements per work-item, corresponding to the N memory locations at src + i * get_max_local_range() + get_local_id() for i between 0 and N.

New:

> Effects: Loads N elements from in_iter to out using the g group object. Properties may specify [data placement](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_oneapi_group_load_store.asciidoc#data_placement). Default data placement is a blocked one: out[i] = in_iter[g.get_local_linear_id() * N + i]; in striped case: out[i] = in_iter[g.get_local_linear_id() + g.get_local_linear_range() * i]; for i between 0 and N.